app/experimenter/nimbus-ui/src/components/PreviewURL/index.tsx

```diff
@@ -53,8 +68,8 @@ export const PreviewURL: React.FC<
         Preview URL
       </h3>
       <p>
-        <b>Click to copy the link</b> and paste it in your browser. You need to
-        enable <code>nimbus.debug</code> pref before.
+        <b>Click the about:studies link below to copy</b> then paste it in your
+        browser. Ensure <code>nimbus.debug</code> about:config pref is enabled.
```

Comment:
Can you make this Ensure you enable `nimbus.debug` in  `about:config` first.

Reply:
Ha. Good thing I opened up this comment in github instead as the email version didn't clearly show the <code>about:config</code> formatting